src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/oops/ConstantPool.java

```diff
@@ -435,26 +436,23 @@ public int getMethodTypeIndexAt(int i) {
     return res;
   }
 
+    private U4Array getOffsets() {
+        Address a =  getAddress().addOffsetTo(bsmaentries);
+        if (a == null) return null;
+        a = bsmaentries_offsets.getValue(a);
+        return VMObjectFactory.newObject(U4Array.class, a);
+  }
```

Comment:
```suggestion
  private U4Array getOffsets() {
     Address a =  getAddress().addOffsetTo(bsmaentries);
     if (a == null) return null;
     a = bsmaentries_offsets.getValue(a);
     return VMObjectFactory.newObject(U4Array.class, a);
  }
```
Fix indent. (Note this file is using  2-indent instead of the 4-indent it should for Java code.)